lib/pizza_bot.rb

```diff
@@ -1,4 +1,5 @@
 class PizzaBot
+  # First round of edits
   attr_reader :width, :height, :max_dimensions, :coordinates, :directions
 
   def initialize(string)
```

Comment:
Feel like i'm doing a lot of janky string parsing but i'm not sure of a better alternative. The instructions say the incoming strings will be all properly formatted so my solution will work even though it seems fragile.